src/Infrastructure/Controller/MyArea/Profile/DeleteProfileController.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Infrastructure\Controller\MyArea\Profile;
+
+use App\Application\CommandBusInterface;
+use App\Application\User\Command\DeleteUserCommand;
+use App\Domain\User\Exception\UserCannotBeDeletedException;
+use App\Infrastructure\Security\AuthenticatedUser;
+use Symfony\Component\HttpFoundation\RedirectResponse;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpKernel\Exception\AccessDeniedHttpException;
+use Symfony\Component\Routing\Annotation\Route;
+use Symfony\Component\Routing\RouterInterface;
+use Symfony\Component\Security\Http\Attribute\IsCsrfTokenValid;
+
+final class DeleteProfileController
+{
+    public function __construct(
+        private CommandBusInterface $commandBus,
+        private AuthenticatedUser $authenticatedUser,
+        private RouterInterface $router,
+    ) {
+    }
+
+    #[Route('/profile/delete', name: 'app_profile_delete', methods: ['DELETE'])]
+    #[IsCsrfTokenValid('delete-user')]
```

Comment:
Comme indiqué hier, tu devrais pouvoir retravailler ce contrôleur pour qu'il gère à la fois les requêtes GET (affichage de la page) et les requêtes DELETE (demande de suppression du compte après avoir cliqué sur le bouton et confirmé via la modale)

(Détail) Renommer le token CSRF pour parler aussi de "profil"

```suggestion
    #[Route('/profile/delete', name: 'app_profile_delete', methods: ['GET', 'DELETE'])]
    #[IsCsrfTokenValid('delete-profile')]
```